src/modules/poweraccent/PowerAccent.UI/Selector.xaml

```diff
@@ -15,60 +15,66 @@
+                    </ItemsPanelTemplate>
+                </ListBox.ItemsPanel>
+                <ListBox.ItemTemplate>
+                    <DataTemplate>
+                        <TextBlock
+                            VerticalAlignment="Center"
+                            FontSize="18"
+                            Text="{Binding}"
+                            TextAlignment="Center" />
+                    </DataTemplate>
+                </ListBox.ItemTemplate>
+                <ListBox.ItemContainerStyle>
+                    <Style x:Name="ItemStyle" TargetType="ListBoxItem">
+
+                        <Setter Property="Template">
+                            <Setter.Value>
+                                <ControlTemplate TargetType="{x:Type ListBoxItem}">
+                                    <Border
+                                        x:Name="myBorder"
+                                        Margin="0"
+                                        Padding="0"
+                                        SnapsToDevicePixels="true"
+                                        Style="{DynamicResource borderContent}">
+                                        <StackPanel Orientation="Vertical">
+                                            <ContentPresenter />
+                                        </StackPanel>
+                                    </Border>
+                                    <ControlTemplate.Resources>
+                                        <Style x:Key="borderContent" TargetType="Border">
+                                            <Setter Property="BorderThickness" Value="0" />
+                                            <Setter Property="BorderBrush" Value="Transparent" />
+                                            <Setter Property="Background" Value="#ECECEC" />
+                                            <Setter Property="Width" Value="50" />
+                                            <Setter Property="Height" Value="50" />
+                                        </Style>
+                                    </ControlTemplate.Resources>
+                                    <ControlTemplate.Triggers>
+                                        <Trigger Property="IsSelected" Value="true">
+                                            <Setter TargetName="myBorder" Property="Background" Value="White" />
+                                            <Setter Property="FontWeight" Value="Bold" />
+                                        </Trigger>
+                                    </ControlTemplate.Triggers>
+                                </ControlTemplate>
+                            </Setter.Value>
+                        </Setter>
+                    </Style>
+                </ListBox.ItemContainerStyle>
+            </ListBox>
+            <TextBlock x:Name="characterName" FontSize="12" Text="(U+0000) A COOL LETTER NAME COMES HERE" Visibility="{Binding characterName}"></TextBlock>
```

Comment:
- Adding `"TextAlignment"="Center"`.
- Wrapping into border element to set border to zero/transparent.

Reply:
In the border element it would get set under every element.

Reply:
Yes.

Reply:
But that's exactly not what I want it to do. Or do I miss something here?

Reply:
My idea regarding the border element was removing the border around the text box. Therefore you have to wrap it into a border element that inherits its property to the text box.

Reply:
Doing this doesn't make any difference